src/transformers/image_utils.py

```diff
@@ -155,6 +156,29 @@ def to_numpy_array(img) -> np.ndarray:
     return to_numpy(img)
 
 
+def replicate_channels(img: np.ndarray, num_channels: int = 3) -> np.ndarray:
```

Comment:
We should be able to specify the output data format with this function

Reply:
You mean the type e.g. np.uint8, np.float?

Reply:
Oh, sorry, I meant have the  input argument `data_format` which will be one of the enum values from ChannelDimension